bgpd/bgp_nexthop.c

```diff
@@ -379,6 +401,8 @@ void bgp_connected_add(struct bgp *bgp, struct connected *ifc)
 		if (IN6_IS_ADDR_LINKLOCAL(&p.u.prefix6))
 			return;
 
+		bgp_address_add(bgp, ifc, addr);
```

Comment:
shouldn't this be before `if (IN6_IS_ADDR_LINKLOCAL(&p.u.prefix6))`? My question is: do we skip LL addresses to be added?

Reply:
Yes, we are not adding LL in this hash table. As i had explained in another comment the behaviour before & after this PR remain the same.